### ISSUE TITLE
Grid select box issue

### ISSUE THREAD

**Issue by hoon-jo** [nhn/tui.grid#1219]
I had to create a new project and i installed toast grid in my new project
that was all fine to develop in different environment.

when i finished my work, i moved my codes to old project. I expected it's going to work because i've been developing in same environment with old project. but it doesn't work and i don't know why

it's supposed to look like down below

![image](https://user-images.githubusercontent.com/65211698/98503280-713a8a00-2297-11eb-99e8-54c4f28d2340.png)

that obviously worked in my new project. 
the select box is working

but in my old project, it looks like this

![image](https://user-images.githubusercontent.com/65211698/98503435-d8f0d500-2297-11eb-836a-e3d861338412.png)

it doesn't work the select box. but i can choose one of them in select box at least
I can't figure out why it looks like that. I checked all dependencies that i have

this is my new project's package.json (the project that works)

```
"dependencies": {
    "@reduxjs/toolkit": "^1.4.0",
    "@testing-library/jest-dom": "4.2.4",
    "@testing-library/react": "9.3.2",
    "@testing-library/user-event": "7.1.2",
    "@toast-ui/react-editor": "1.0.1"...
    "@toast-ui/react-grid": "^4.15.0",
    "antd": "3.21.2",
    "axios": "0.19.2",
    "file-saver": "^2.0.2",
    "lodash": "^4.17.20",
    "moment": "2.27.0",
    "ramda": "^0.27.1",
    "ramda-adjunct": "^2.27.0",
    "react": "16.13.1",
    "react-dom": "16.13.1",
    "react-moment": "^0.9.7",
    "react-redux": "^7.2.1",
    "react-router-dom": "^5.2.0",
    "react-scripts": "3.4.1",
    "recharts": "^1.8.5",
    "styled-components": "4.3.2",
    "xlsx": "^0.16.6",
    "xlsx-populate": "^1.21.0"
  },
...
```

and there is my old project's package.json down below (select box doesn't look properly)

```
"dependencies": {
    "@ant-design/icons": "^4.2.2",
    "@reduxjs/toolkit": "^1.4.0",
    "@toast-ui/react-editor": "1.0.1",
    "@toast-ui/react-grid": "^4.15.1",
    "ant-design-pro": "2.3.2",
    "antd": "3.21.2",
    "axios": "0.19.0",
    "babel-plugin-import": "1.12.2",
    "chart.js": "2.8.0",
    "chartjs-plugin-annotation": "0.5.7",
    "cors": "2.8.5",
    "customize-cra": "0.8.0",
    "draft-js": "0.11.0",
    "draft-js-import-html": "1.3.3",
    "draftjs-to-html": "0.8.4",
    "env-cmd": "10.0.1",
    "express": "4.17.1",
    "file-saver": "2.0.2",
    "less": "3.10.3",
    "less-loader": "5.0.0",
    "lodash": "^4.17.20",
    "moment": "2.24.0",
    "moment-range": "4.0.2",
    "nodemon": "1.19.4",
    "normalize.css": "8.0.1",
    "prop-types": "15.7.2",
    "ramda": "0.26.1",
    "ramda-adjunct": "^2.28.0",
    "react": "16.8.6",
    "react-app-polyfill": "1.0.5",
    "react-app-rewired": "2.1.5",
    "react-async": "^10.0.1",
    "react-big-calendar-like-google": "0.0.3",
    "react-chartjs-2": "2.7.6",
    "react-color": "2.17.3",
    "react-dom": "16.8.6",
    "react-draft-wysiwyg": "1.13.2",
    "react-moment": "^1.0.0",
    "react-redux": "^7.2.1",
    "react-router-dom": "5.2.0",
    "react-scripts": "3.0.1",
    "socket.io-client": "2.2.0",
    "styled-components": "4.3.2",
    "tui-date-picker": "4.0.3",
    "tui-grid": "4.7.0",
    "xlsx": "0.15.2",
    "xlsx-populate": "^1.21.0"
  },
...
```

pls somebody tell me why that look like that

**Comment by js87zz** [comment]
@hoon-jo 
We cannot resolve the issue exactly using your information. 
First, update your tui-grid version in your old env.

**Comment by js87zz** [comment]
@hoon-jo 
The select box height has been fixed in `v4.17.0`. Thanks!
